CHANGELOG.md

```diff
@@ -1,11 +1,16 @@
 # Changelog
 
-## TBD
+## 1.1.0 (2021-12-06)
 
 ### Enhancements
 
 * Backport `persistenceDirectory` config option to Android [bugsnag-cocos2dx#6](https://github.com/bugsnag/bugsnag-cocos2dx/pull/6)
 
+### Bug fixes
+
+* (Android) Bump the bugsnag-android dependency version to v4.22.3
+* (Android) Fix UTF-8 encoding errors in Bugsnag::notify [bugsnag-android#1542](https://github.com/bugsnag/bugsnag-android/pull/1542)
```

Comment:
Let's do it in the usual format. I believe it's just a single patch update?

```suggestion
* (Android) Fix UTF-8 encoding errors in Bugsnag::notify [bugsnag-android#1542](https://github.com/bugsnag/bugsnag-android/pull/1542)
* Update bugsnag-android to v4.22.3
  * Allow disabling previous signal handler invocation for Unity ANRs [bugsnag-android#743]
  * Avoid polling when detecting ANRs by invoking JNI from SIGQUIT handler [bugsnag-android#741]
```

Reply:
Side note - but it'd be good to define what the usual changelog format is sometime. I went with the existing structure in this changelog but agree this makes more sense.